Strassens_Algorithm/Strassens_Algorithm.py

```diff
@@ -0,0 +1,70 @@
+#Strassen's matrix multiplication Algorithm for NxN matrix
+
+#Function to form sub matrices
+def split_matrix(a):
+    matrix_length = len(a)
+    mid = matrix_length / 2
```

Comment:
Do not change // to / . mid should be integer to be used as an argument to range()

Reply:
ohh yeah ,I'll correct it. i mistook that for the resolving issue

Reply:
Sure